flows/triggers/flow_action.go

```diff
@@ -77,7 +77,7 @@ var _ flows.TriggerWithRun = (*FlowActionTrigger)(nil)
 
 type flowActionTriggerEnvelope struct {
 	baseTriggerEnvelope
-	RunSummary json.RawMessage `json:"run_summary"`
```

Comment:
You get a panic if this field isn't included - mailroom only creates flow_action triggers with this set